analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/ComplexProjectFunctionalTest.java

```diff
@@ -98,11 +100,10 @@ public void ensureAlignmentFileCreated()
                         .filteredOn(t -> !getVersion(t).contains("redhat"))
                         .satisfies(l -> {
                             // make sure the two dynamic dependencies not aligned exist and have versions similar to what we expect
-
                             assertThat(l).filteredOn(t -> "HdrHistogram".equals(getArtifactId(t)))
-                                    .hasOnlyOneElementSatisfying(t -> assertThat(getVersion(t)).startsWith("2."));
+                                    .singleElement(as(STRING)).contains(":2.");
                             assertThat(l).filteredOn(t -> "commons-lang3".equals(getArtifactId(t)))
-                                    .hasOnlyOneElementSatisfying(t -> assertThat(getVersion(t)).startsWith("3."));
+                                    .singleElement(as(STRING)).contains(":3.");
```

Comment:
Slight change here, but maybe there's a different away to do it (or it doesn't make much of a difference).

Reply:
Will fix.